File: FireDrill/JavaScript code/TaskNine.js
// TaskNine

let total = 0;
for(let number = 1; number <= 10; number++) {
    if (number % 4 == 0) {
	let sum = 0;
      for(let numbers = 1; numbers <= 5; numbers++) {
	 sum += Math.floor(Math.pow(number, numbers));
}
	total += sum;
}
}
console.log(total * total);

File: FireDrill/Java code/TaskNine.java
public class TaskNine {
	public static void main(String[] args) {

	int total = 0;
	for(int number = 1; number <= 10; number++) {
	    if (number % 4 == 0) {
		int sum = 0;
	      for(int numbers = 1; numbers <= 5; numbers++) {
		 sum += (int)Math.pow(number, numbers);
}
	total += sum;
}
}
	System.out.print(total * total + " ");
}
}